### PR TITLE
Jomefavourite/issue#833

### PR DESCRIPTION
**Pull request by jomefavourite**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
@jomefavourite you need to select the `layer5-ng` branch to create the PR against. I am adding a SS of how to do so. 
Screenshot from 2020-10-01 21-54-44

**Comment by jomefavourite** [comment]
Okay 🤧, but I went to layer5-ng to make the pull request.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Oops. Wrong branch? Intended for `layer5-ng`, I suspect.

**Comment by jomefavourite** [comment]
Changed now @Tanuj22 
Hope it all done and perfect now, if not someone will have to show me how to push the changes correctly pls

**Comment by Jashpatel1** [comment]
> Changed now @Tanuj22
> Hope it all done and perfect now, if not someone will have to show me how to push the changes correctly pls

Looks good to me. @Tanuj22

File: src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
```
Comment (Tanuj22) [inline src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:2]: Remove this

File: src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+type: news
```
Comment (Tanuj22) [inline src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:3]: Remove this

File: src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+author: the Newsroom
+date: 2019-12-27 08:04:00 +0000
+title: "Vendors make a splash in 2019 service mesh implementation rush"
+image: ./2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush.png
```
Comment (Tanuj22) [inline src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:7]: change `image` to `thumbnail`

File: src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#permalink: https://searchapparchitecture.techtarget.com/feature/Vendors-make-a-splash-in-2019-service-mesh-implementation-rush
```
Comment (Tanuj22) [inline src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:8]: Remove this

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
The frontmatter should be like existing news articles present. I have commented on what should be changed. Also, can you finally open a new PR (really really sorry for this) so that we look at the netlify deployment?

**Comment by jomefavourite** [comment]
> The frontmatter should be like existing news articles present. I have commented on what should be changed. Also, can you finally open a new PR (really really sorry for this) so that we look at the netlify deployment?

😭😭😭

File: src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx (hunk 4 above)
Comment (jomefavourite) [reply src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:8]: So I should remove all selected and not put anything there right? Just to be very clear

Comment (Tanuj22) [reply src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:8]: Yes

**Comment by Tanuj22** [comment]
done in #887

**Pull request closed without merging**